### ISSUE TITLE
Please add Labels

### ISSUE THREAD

**Issue by mgifford** [isaacs/github#1188]
@isaacs I added this repo a while back but haven't really been maintaining it https://github.com/mgifford/GitHub4All/

Would be nice to see labels so that we can assign issues based on the type of feature requested. For instance https://github.com/isaacs/github/issues/1182 should be tagged with accessibility.

**Comment by cirosantilli** [comment]
Isaacs stopped replying a while ago, and new admins cannot be added.

The only solution is to give us money :-)
